### PR TITLE
move to spring zeebe 8.5.1

### PR DESCRIPTION
**Pull request by jonathanlukas**
## Related issues

closes #

### CODE REVIEW

File: connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/testutil/command/PublishMessageCommandDummy.java [hunk 1]
```diff
@@ -41,6 +41,11 @@ public PublishMessageCommandStep3 correlationKey(String correlationKey) {
     return this;
   }
 
+  @Override
+  public PublishMessageCommandStep3 withoutCorrelationKey() {
+    return null;
```
Comment (johnBgood) [inline connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/testutil/command/PublishMessageCommandDummy.java:46]: shouldn't this be `return this;` for consistency ?

Comment (chillleader) [reply connector-runtime/connector-runtime-core/src/test/java/io/camunda/connector/runtime/core/testutil/command/PublishMessageCommandDummy.java]: Lol good catch

**Review by johnBgood — APPROVED** [review]
👍

**Comment by chillleader** [comment]
I created another PR from my own branch: https://github.com/camunda/connectors/pull/2351
I also included the config properties changes there, and targeted it on `main`, with a pending backport to `release/8.5` (that would be the correct way to get it into the release branch).

**Pull request closed without merging**